<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscreteContinousSystemDialog</class>
 <widget class="QDialog" name="DiscreteContinousSystemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditZController">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>30</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;give Name of Expression (Transferfunction of Complex Variable z)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLSystem">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>80</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;give Name of Expression (Transferfunction of Complex Variable s)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../ressource.qrc">:/pic/Pictures/DiscreteContinousSys.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Controller (Z-Domain)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>System (Laplace-Domain)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShowKernel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show kernel of continous system</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxDt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sampling interval for discretizing the Continous Transferfunction&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDots">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of dot for discretizing the Continous Transferfunction&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>57</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Num of Dots</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonUpdate">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ressource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiscreteContinousSystemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiscreteContinousSystemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
